src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -49,6 +51,10 @@
 	$: currentPicture =
 		data.pictures.find((picture) => picture._id === $page.url.searchParams.get('picture')) ??
 		data.pictures[0];
+	const googlePicture =
+		data.pictures.find((picture) => {
+			return picture.storage.formats.find((image) => image.width > 500 && image.height > 500);
+		}) ?? data.pictures[0];
```

Comment:
ok no actually `currentPicture` should be used always

BUT the correct format should be displayed - the smallest one that has both >= 500 width & height